pyproject.toml

```diff
@@ -77,8 +77,8 @@ doc = [
   "ipyleaflet",
   "colorama",
 ]
-test = ["pytest", "pytest-cov", "pytest-regressions"]
-dev = ["pyyaml", "pre-commit", "nox", "pydata-sphinx-theme[doc,test]"]
+test = ["pytest", "pytest-cov", "pytest-regressions", 'sphinx[test]']
```

Comment:
@trallard FYI adding `[test]` to sphinx is the main change I pushed to get the defusexml error message to go away.

Reply:
were you seeing that message in CIs or locally?  I thought after #1772 the CIs should not complain about that anymore...

Reply:
Since this PR removes the github_install.sh the test dependencies needed adding indeed.

Reply:
Regardless I think it make sens for `[test]` to depends on `sphinx[test]` :-)